### DISCUSSION TITLE
Animation for drawer-side retraction

### DISCUSSION THREAD

**Discussion by kristley** [saadeghi/daisyui#1959] [Q&A] [answered]
How can I make an retraction animation for drawer-side in daisy 3.0? It used to have a retraction animation by default in 2.x, but not anymore it seems. [drawer-side v2.0](https://v2.daisyui.com/components/drawer/#:~:text=%23-,Drawer,-Preview)

**Comment by OSAMA263** [comment, marked as answer]
i think there is a better way but thats what i have for now
put this in ur index.css

`
.drawer-side > *:not(.drawer-overlay) {
  grid-row-start: 1;
  grid-column-start: 1;
  transition: all 0.4s;
}
`

**Reply by kristley** [reply]
That worked perfectly! Thank you so much!
